drink/src/runtime.rs

```diff
@@ -39,14 +39,6 @@ pub trait Runtime: frame_system::Config {
 
     /// Default actor for the runtime.
     fn default_actor() -> AccountIdFor<Self>;
-
-    /// Metadata of the runtime.
-    fn get_metadata() -> RuntimeMetadataPrefixed;
-
-    /// Convert an account to an call origin.
-    fn convert_account_to_origin(
-        account: AccountIdFor<Self>,
-    ) -> <<Self as frame_system::Config>::RuntimeCall as Dispatchable>::RuntimeOrigin;
```

Comment:
Removed these 2 as , (as far as I can tell) there are no easy way to implement them directly with the where clause. But they can be easily called on the concrete type directly if needed later on

Reply:
ah looks like this might be problematic as it's used in the drink integration